.github/workflows/R-CMD-check.yaml

```diff
@@ -12,6 +12,10 @@ on:
   merge_group:
   workflow_dispatch:
 
+  concurrency:
```

Comment:
This groups together workflow runs and will cancel one if another on the same branch is triggered, or something like this?

Reply:
yes exactly